integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/passthru/transport/test/PassthroughTransportHttpProxyTestCase.java

```diff
@@ -34,46 +32,32 @@
 
 public class PassthroughTransportHttpProxyTestCase extends ESBIntegrationTest {
     private ServerConfigurationManager serverConfigurationManager;
-    private LogViewerClient logViewer;
 
     @BeforeClass(alwaysRun = true)
     public void setEnvironment() throws Exception {
-        super.init();
-        serverConfigurationManager = new ServerConfigurationManager(
-                new AutomationContext("ESB", TestUserMode.SUPER_TENANT_ADMIN));
-        serverConfigurationManager.applyConfiguration(new File(
+        serverConfigurationManager = new ServerConfigurationManager(new AutomationContext());
+        serverConfigurationManager.applyMIConfiguration(new File(
                 getESBResourceLocation() + File.separator + "passthru" + File.separator + "transport" + File.separator
                         + "httpproxy" + File.separator + "axis2.xml"));
         super.init();
-        loadESBConfigurationFromClasspath("/artifacts/ESB/passthru/transport/httpproxy/httpProxy.xml");
-        logViewer = new LogViewerClient(contextUrls.getBackEndUrl(), getSessionCookie());
     }
 
     @Test(groups = "wso2.esb", description = "Passthrough Transport Http.proxy test case")
     public void passthroughTransportHttpProxy() throws Exception {
-        int beforeLogSize = logViewer.getAllSystemLogs().length;
+        CarbonLogReader carbonLogReader = new CarbonLogReader();
+        carbonLogReader.start();
 
         try {
-            axis2Client.sendSimpleStockQuoteRequest(getProxyServiceURLHttp("HttpProxyTest"), "", "IBM");
+            axis2Client.sendSimpleStockQuoteRequest(getProxyServiceURLHttp("PassthroughTransportHttpTestProxy"), "", "IBM");
         } catch (AxisFault expected) {
             //read timeout expected
         }
-        LogEvent[] logs = logViewer.getAllSystemLogs();
-        int afterLogSize = logs.length;
-
-        boolean proxyhostEntryFound = false;
-        for (int i = 0; i < (afterLogSize - beforeLogSize); i++) {
-            if (logs[i].getMessage().contains("111.wso2.com:7777")) {
-                proxyhostEntryFound = true;
-                break;
-            }
-        }
-        assertTrue(proxyhostEntryFound);
+        assertTrue(carbonLogReader.assertIfLogExists("111.wso2.com:7777"));
```

Comment:
Shall we add an assert message here?

Reply:
Fixed in https://github.com/wso2/micro-integrator/pull/580